cybersecuritytools/csls/generate_cloudwatch_logs/put_cloudwatch_logs.py

```diff
@@ -0,0 +1,154 @@
+"""Module for generating test data of various types, send data to Cloudwatch."""
+import json
+from dataclasses import dataclass
+from datetime import datetime, timezone
+from typing import Any, Dict, List
+from uuid import uuid4
+
+import boto3
+import mypy_boto3_logs as logs
+from mypy_boto3_logs.type_defs import PutLogEventsResponseTypeDef
+
+
+class LogEventResponseReal(PutLogEventsResponseTypeDef):
+    ResponseMetadata: Dict[Any, Any]
+
+
+def logs_client() -> logs.CloudWatchLogsClient:
+    return boto3.client("logs")
```

Comment:
This now has different behaviour, each call to `logs_client` creates a new boto client. While this is fine, why not just create it once?

Reply:
It doesn't affect the performance of the task either way. The lru_cache is a non obvious addition for no real world improvement.

Reply:
Is that so? I would have assumed one call to `boto3.client("logs")`, which is then stored in a variable, would be better than many calls to `boto3.client("logs")`.